src/middlewared/middlewared/etc_files/local/nut/ups_config.py

```diff
@@ -16,7 +16,7 @@ def generate_ups_config(middleware):
 
     ups_group = middleware.call_sync('group.query', [['group', '=', UPS_USER]], {'get': True})
     os.chown(UPS_VARPATH, 0, ups_group['gid'])
-    os.chmod(UPS_VARPATH, 0o770)
+    os.chmod(UPS_VARPATH, 0o775)
```

Comment:
What is impact of making this world-readable? Have you checked for what other files nut may write there?

Reply:
@anodos325 we have `upssched.pipe` there apart from the pid file, i have adjusted its perms to be 660 and we should be good hopefully..testing the changes to make sure there are no regressions from this change

Reply:
@anodos325 my latest changes are good, confirmed after testing ✅

Reply:
Maybe we should configure a separate rundir for upssched? It looks configurable via PIPEFN / LOCKFN.